### ISSUE TITLE
Add Support for Server Type: Flood (Node.JS)

### ISSUE THREAD

**Issue by KangarooCreativeTeam** [bogenpirat/remote-torrent-adder#190]
Hiya, 

For those that do not know, there is a new frontend to rTorrent called [Flood](https://github.com/jfurrow/flood); _"A modern web UI for rTorrent with a Node.js backend and React frontend."_

To put simply, it is the bee's knees.  It's really sexy, and performs much better than RUTorrent (I have upwards of a thousand torrents active (in some form or another) at any one time = working far better, & lighter than RUTorrent).  
It is just so damn awesome, I've completely switched over from RUTorrent after my first play around with it (using it over 2 months now = so happy ^_^).  

Now, we need to add support for it to the awesome Remote-Torrent-Adder chrome extension for a more awesome experience out in the vastness of the internet's torrent jungles. </overly_dramatic>

I did try to make it work with the existing providers, but couldn't get it working...
Note; It's got a NODE.JS backend - I could find out more specific info from the main author if needed. 

Thank you

TL;DR

Add support for Server Type: FLOOD  (Node.JS)

**Comment by bogenpirat** [comment]
i'll take a look. at first glance it doesn't seem like you can supply a label upon adding a torrent, only a destination directory. i'll leave the interactive selection dialog out for now.

**Comment by bogenpirat** [comment]
5df4ef0 adds support for:

- adding torrents
- adding magnets
- static directory setting
- static start paused setting

if you could give it a go, there are instructions on how to test it [here](https://github.com/bogenpirat/remote-torrent-adder/wiki/Try-new-code), use this [zip](https://github.com/bogenpirat/remote-torrent-adder/archive/master.zip).

**Comment by KangarooCreativeTeam** [comment]
Hiya, 

Okay, all tested! - I can confirm everything is working great!  ^_^

_NOTE; the only option I have not tested is the SSL, as I'm not set up for that._

I have tested every (other) option, adding magnets, adding torrents, both paused and unpaused. 

I tested the optional directory - I did as specified; an absolute path under my default (rTorrent) download directory - it worked as expected and put new downloads in the specified sub-directory (it was created automatically - I did not have to create it myself (just mentioning this for completeness)).

I did have to specify my username & pw in the config for the magnet/torrent to be added, which is expected as I am using uname/pw for my installation of FLOOD (works as expected). 

I'm also using a custom port (works as expected)

Also, if relevant; specifically, I tested by adding linux isos' via torrentproject.se, zooqle.com, and thepiratebay.org (all working as expected)

So; everything works great (only SSL is untested). 

-----------------------------------------------------------------------

Thank you so much for such a fast turnaround.  You have my respect.  And thanks for RTA - it really makes adding magnets/torrents so much less painful - Thank you!

^_^

**Comment by bogenpirat** [comment]
alright, good work. i'll release tomorrow or so.
